electron/src/electron/network.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  WebPlugin, NetworkPlugin, NetworkStatus
+} from "@capacitor/core";
+
+export interface PluginListenerHandle {
+  remove: () => void;
+}
+
+declare var window:any;
+
+export class NetworkPluginElectron extends WebPlugin implements NetworkPlugin {
+
+  listenerFunction:any = null;
+
+  constructor() {
+    super({
+      name: 'Network',
+      platforms: ['electron']
+    });
+  }
+
+  getStatus(): Promise<NetworkStatus> {
+    return new Promise((resolve, reject) => {
+      if(!window.navigator)
+        reject('Network info not available');
+      let connected = window.navigator.onLine;
+      resolve({connected: connected, connectionType: connected ? 'unknown' : 'none'});
```

Comment:
Can we use the connection type here if available? https://developer.mozilla.org/en-US/docs/Web/API/Network_Information_API

Reply:
The current stable version of electron does not have `navigator.connection` implemented as it runs on `Chrome v59` and `navigator.connection` is only in `Chrome v61 +`. The master branch in electron is in beta for `2.0.0` and this will bring `Chrome v61` so I will update all of this when `2.0.0` hits stable